### DISCUSSION TITLE
Enhanced OpenAI-Whisper-GUI

### DISCUSSION THREAD

**Discussion by meeksqueal** [openai/whisper#1578] [Show and tell]
I quickly enhanced the python OpenAI-Whisper-GUI from @iamironman0.

- Added the option to save the translation/transcription to a .txt, .srt, .vtt, .tsv, or .json file.
- Added the option to directly save the translation/transcription to the input video, either as hardcoded subtitles or embedded in the .mkv format.

https://github.com/meeksqueal/OpenAI-Whisper-GUI

**Comment by rudymohammadbali** [comment]
@meeksqueal good job👍, I will try add these features to the next version of the app.

**Comment by ViennaHarleking** [comment]
Hi, would love to try it, but it doesn't work in my case:
`/home/harleking/PythonCode/transgui/OpenAI-Whisper-GUI-main/openai/lib/python3.10/site-packages/whisper/timing.py:58: NumbaDeprecationWarning: The 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.
  def backtrace(trace: np.ndarray):
Traceback (most recent call last):
  File "/home/harleking/PythonCode/transgui/OpenAI-Whisper-GUI-main/main.py", line 580, in <module>
    app = WhisperGui()
  File "/home/harleking/PythonCode/transgui/OpenAI-Whisper-GUI-main/main.py", line 538, in __init__
    self.iconbitmap(app_logo)
  File "/home/harleking/PythonCode/transgui/OpenAI-Whisper-GUI-main/openai/lib/python3.10/site-packages/customtkinter/windows/ctk_tk.py", line 230, in iconbitmap
    super().wm_iconbitmap(bitmap, default)
  File "/usr/lib/python3.10/tkinter/__init__.py", line 2109, in wm_iconbitmap
    return self.tk.call('wm', 'iconbitmap', self._w, bitmap)
_tkinter.TclError: bitmap "./icons/logo.ico" not defined`

And i cant help myself because i am noob at programming.

**Reply by rudymohammadbali** [reply]
@ViennaHarleking make sure you have "logo.ico" inside "./icons/" directory.
